tests/framework/installer/ceph_installer.go

```diff
@@ -910,7 +923,7 @@ func (h *CephInstaller) waitForCleanupJobs(namespace string) error {
 	}
 
 	logger.Info("waiting for job(s) to cleanup the host...")
-	err := wait.Poll(5*time.Second, 90*time.Second, allRookCephCleanupJobs)
+	err := wait.Poll(5*time.Second, 180*time.Second, allRookCephCleanupJobs)
```

Comment:
I would expect the cleanup to happen under 90s. Did you see it take longer than that? I'd be curious to see the logs.

Reply:
mgr pod takes more time to come, so having 90 was leading to error and 180 was taken from [here](https://github.com/rook/rook/pull/3523/files#diff-7d86439d1976937dad3f5a30d519f239R750) but when I change the `RETRY_MAX` to 40 with `90*time.Second` mgr pod is up.

Also, we already have this `RETRY_MAX`  as env var go we just need to export it with 40 by default keeping 30. I added this in doc.

Reply:
ok that works for now, the more we can understand the delays in the tests, the more efficient we can hope to make them